src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -1,7 +1,7 @@
 <template>
 	<www-page class="lend-by-category-page">
 		<div class="heading-region">
-			<h1>Make a loan, change a life</h1>
+			<h1 @click="isAdmin=!isAdmin">Make a loan, change a life</h1>
```

Comment:
what's this toggle for?

Reply:
I changed isAdmin to default to false so that the admin tool will not display. But we don't actually check if a user is an admin yet, so I added this hidden toggle as a temporary way to be able to see the tool.